healthcareai/tests/test_deploy_supervised_model_class.py

```diff
@@ -120,3 +121,22 @@ def runTest(self):
 
     def tearDown(self):
         del self.o
+
+
+class TestValidateDestinationTableConnection(unittest.TestCase):
+    def test_raises_error_on_table_not_existing(self):
+        try:
+            result = validate_destination_table_connection('localhost', 'foo', 'bar', 'baz')
+            # Fail the test if the above call doesn't throw an error
+            self.fail()
+        except HealthcareAIError as e:
```

Comment:
I understand why you're doing this test this way - you want to test the exception message matches what you want it to say.  I would reconsider putting constraints this strict into the unit test.  It's a matter of preferences, but it doesn't feel right to break the build just because the wording of an exception was changed.

Instead, I would only assert that the expected type of exception was raised and call it a day.